test/fuzz/.config.json.example

```diff
@@ -1,19 +1,33 @@
 {
 	"transitionWeights": {
 		"CreateTopic": 2,
-		"FundTopic": 10,
+		"FundTopic": 8,
 		"RegisterWorker": 4,
 		"RegisterReputer": 4,
-		"StakeAsReputer": 10,
-		"DelegateStake": 10,
-		"CollectDelegatorRewards": 10,
-		"DoInferenceAndReputation": 30,
+		"StakeAsReputer": 6,
+		"DelegateStake": 6,
+		"CollectDelegatorRewards": 6,
+		"DoInferenceAndReputation": 20,
 		"UnregisterWorker": 4,
 		"UnregisterReputer": 4,
 		"UnstakeAsReputer": 6,
 		"UndelegateStake": 6,
 		"CancelStakeRemoval": 0,
-		"CancelDelegateStakeRemoval": 0
+		"CancelDelegateStakeRemoval": 0,
+		"AddToAdminWhitelist": 0,
```

Comment:
I am ok with having these 0 by default but I would prefer them to at least happen 1% of the time